### PR TITLE
Fixes #4609 - improve class import output

### PR DESCRIPTION
**Pull request by tstrachota**

### CODE REVIEW

File: test/functional/proxy_test.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      }
+    end
+
+    it 'prints detailed report' do
+      api_expects(:smart_proxies, :import_puppetclasses, 'Import classes')
+        .with_params('id' => '83')
+        .returns(report)
+
+      output = OutputMatcher.new([
+        'Result:               Successfully updated environment and puppetclasses from the on-disk puppet installation',
```
Comment (mbacovsky) [inline test/functional/proxy_test.rb]: Can we do something about the gap between `Result:` and the message? I probably understand where it comes from but here it doesn't make much visual sense. Any idea?
```
Result:               Successfully updated environment and puppetclasses from the on-disk puppet installation
Changed environments: 
 1) production
    New classes: 
     1) apt::backports
     2) apt
     3) apt::params
```
Can we change the numbering for an unnumbered list? In my testing imports I had more then hundred new classes and the numbering does not improve readability.

**Comment by tstrachota** [comment]
@mbacovsky I removed numbering and fixed the message for no changes. The only thing that we can currently do about the message indentation is to put it on a new line. Check if you like it more.

**Comment by mbacovsky** [comment]
A way better now! Thanks @tstrachota. :poodle:  
```
Result:               
  Successfully updated environment and puppetclasses from the on-disk puppet installation
Changed environments: 
 1) production
    New classes: 
        elasticsearch::config
        elasticsearch
        elasticsearch::package
        elasticsearch::repo
```
It is good to go when the tests are green.

**Comment by tstrachota** [comment]
@mbacovsky test failures are unrelated (http://projects.theforeman.org/issues/22964). Could you merge this?

**Comment by mbacovsky** [comment]
[test]

**Comment by mbacovsky** [comment]
:+1: This is good to go now! Thanks..

**Pull request merged**